lib/taskana-core/src/test/java/acceptance/task/CompleteTaskAccTest.java

```diff
@@ -58,13 +101,12 @@ void should_completeClaimedTaskByAnotherUser_When_UserIsAdmin() throws Exception
   @WithAccessId(user = "admin")
   @WithAccessId(user = "taskadmin")
   @TestTemplate
-  void should_ForceCompleteTask_When_NoExplicitPermissionsButUserIsInAdministrativeRole()
+  void should_ForceCompleteClaimedTask_When_NoExplicitPermissionsButUserIsInAdministrativeRole()
       throws Exception {
-    resetDb(false);
+    Task task = createTaskClaimedByUser();
+
+    Task completedTask = taskService.forceCompleteTask(task.getId());
 
-    assertThat(taskService.getTask("TKI:000000000000000000000000000000000000").getState())
-        .isEqualTo(TaskState.CLAIMED);
-    Task completedTask = taskService.forceCompleteTask("TKI:000000000000000000000000000000000000");
     assertThat(completedTask).isNotNull();
```

Comment:
Same thing as above: 

> I know you didn't modify this, but maybe you're willing to make an adjustment here ^^
Instead of asserting modified is not created I'd personally assert that completed = modify != null != created.